### PR TITLE
fix VPP ipsec docs & add troubleshooting info

### PR DESCRIPTION
**Pull request by sknat**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico/getting-started/kubernetes/vpp/ipsec.mdx [hunk 1]
```diff
@@ -35,12 +35,34 @@ kubectl -n calico-vpp-dataplane create secret generic calicovpp-ipsec-secret \
 
 ### Configure the VPP dataplane
 
-To enable IPsec, you need to configure two environment variables on the `calico-vpp-node` pod. You can do so with the following kubectl command:
+To enable IPsec, you need first need to enable the ipsec feature in the `calico-vpp-config` ConfigMap.
```
Comment (bmckercher123) [inline calico/getting-started/kubernetes/vpp/ipsec.mdx]:
```suggestion
To enable IPsec, you first need to enable the ipsec feature in the `calico-vpp-config` ConfigMap.
```

File: calico/getting-started/kubernetes/vpp/ipsec.mdx [hunk 2]
```diff
@@ -50,13 +72,95 @@ Once IPsec is enabled, all the traffic that uses IP-in-IP encapsulation in the c
...
+kubectl -n calico-vpp-dataplane exec calico-vpp-node-XXXX  -c vpp -- vppctl
 ```
 
 Then at the `vpp#` prompt, you can run the following commands:
 
 - `show ikev2 profile` will list the configured IKEv2 profiles, there should be one per other node in your cluster
-- `show ipsec sa` will list the establish IPsec SA, two per IKEv2 profile
-- `show interface` will list all the interfaces configured in VPP. The ipip interfaces (which correspond to the IPsec tunnels) should be up
+
+As an example on a three node cluster
```
Comment (bmckercher123) [inline calico/getting-started/kubernetes/vpp/ipsec.mdx]:
```suggestion
As an example on a three-node cluster
```

**Review by bmckercher123 — CHANGES_REQUESTED** [review]
Thanks for your contributions to VPP docs. A few nit edits, then we'll need to get a Dev review.

**Comment by onong** [comment]
CC @lwr20

**Comment by sknat** [comment]
Thanks @bmckercher123 for the review 
and sorry for the delayed reply.

Just updated the PR with your suggested edits

**Review by bmckercher123 — APPROVED** [review]
Approved.

**Pull request merged**